doc/api-extensions.md

```diff
@@ -2873,3 +2873,10 @@ The webhook data matches what's sent over the existing events API.
 ## `storage_driver_truenas`
 
 This adds a TrueNAS storage driver.
+
+## `container_disk_tmpfs`
+
+This adds tmpfs support for disk devices.
+
+* `source=tmpfs:` mounts a tmpfs filesystem, respecting size, uid, gid and mode options
```

Comment:
Should be "file system" and, then size, uid,  gid and mode should all be between "``" to avoid validation errors with the doc.